notify/impl.go

```diff
@@ -747,6 +747,14 @@ func (n *Slack) Notify(ctx context.Context, as ...*types.Alert) (bool, error) {
 				Text:  tmplText(action.Text),
 				URL:   tmplText(action.URL),
 				Style: tmplText(action.Style),
+				Name:  tmplText(action.Name),
+				Value: tmplText(action.Value),
+				ConfirmField: &config.SlackConfirmationField{
```

Comment:
You need to test if `action.ConfirmField` is nil otherwise it will panic.

Reply:
done